python/setup.py

```diff
@@ -297,6 +297,10 @@ def _run_cmake(self):
                     move_shared_libs(build_prefix, build_lib, "plasma")
                 if self.with_gandiva:
                     move_shared_libs(build_prefix, build_lib, "gandiva")
+                    bytecode_files = glob.glob(pjoin(build_prefix, '*.bc'))
```

Comment:
For the record, they are [_bitcode_](https://llvm.org/docs/BitCodeFormat.html) files, not _bytecode_.

Reply:
I'll rename :)

Reply:
Oh, we need to rename "bytecode" in Gandiva. @pravindra

Reply:
I guess bc can mean both :-). I'll fix any references to byte code in gandiva code.